MobileBuy/sample/src/shopify/java/com/shopify/sample/SampleApplication.java

```diff
@@ -30,6 +30,9 @@
 import com.shopify.sample.domain.usecases.UseCases;
 import com.shopify.sample.domain.usecases.UseCasesImpl;
 import com.shopify.sample.util.CallbackExecutors;
+
+import org.jetbrains.annotations.NotNull;
```

Comment:
Do we need this annotation?

Reply:
I dont think so... removing on next commit. THanks!